app/scripts/lib/transform.js

```diff
@@ -40,6 +40,36 @@ define(function (require, exports, module) {
       }
 
       return result.value;
+    },
+    /**
+     * Simplifies user agent operating system names (50+) to generic popular names (~6)
+     *
+     * @param {String} os Operating System name from
+     * @returns {String} generic operating system name
+     */
+    toGenericOSName (os) {
+      if (/^Windows/.test(os)) {
+        return 'Windows';
+      }
+
+      if (/^Android/.test(os)) {
+        return 'Android';
+      }
+
+      if (/^Mac OS/.test(os)) {
+        return 'macOS';
+      }
+
+      if (/^iOS/.test(os)) {
+        return 'iOS';
+      }
+
+      if (/^Ubuntu/.test(os) || /^Linux/.test(os) || /^Fedora/.test(os)
+        || /^Red Hat/.test(os) || /^Debian/.test(os)) {
+        return 'Linux';
+      }
+
+      return 'Unknown';
```

Comment:
Oh, similar comment to the I just left in your auth server PR, do we need to include `'Unknown'` in the localised strings?